app/mailers/tasks_mailer.rb

```diff
@@ -0,0 +1,15 @@
+class TasksMailer < ApplicationMailer
+  default template_path: 'mailers/tasks'
+
+  def task_completed(task, current_user)
+    @task = task
+    @user = current_user
+    watchers = @task.watchers.map {|watcher| watcher.email if watcher != current_user}
```

Comment:
not good, one of watchers emails may be nil
try to use binary subtraction, like: `@task.watchers.map(&:email) - [current_user.email]`

Reply:
also, name variable better, as it contains a collection of emails, not watcher instances